modules/priors/R/priors.R

```diff
@@ -77,72 +86,80 @@ fit.dist <- function(trait.data, trait = colnames(trait.data),
 ##'                 x=c(2, 6, 3.3), 
 ##'                 alpha = 0.05, 
 ##'                 distn = 'lnorm')$optim$bestmem
-#--------------------------------------------------------------------------------------------------#
 prior.fn <- function(parms, x, alpha, distn, central.tendency = NULL, trait = NULL) {
-  if(!distn %in% c('lnorm', 'gamma', 'weibull', 'beta')){
+  if (!distn %in% c("lnorm", "gamma", "weibull", "beta")) {
```

Comment:
This function could probably be replaced by or rewritten to use another package like fitdistplus or rrRiskDistributions that are way more comprehensive and general. I use those packages now when fitting priors so ... maybe this can be deprecated